controllers/controllers.technicians.js

```diff
@@ -20,3 +21,15 @@ exports.getTechnician = async (req, res, next) => {
     next(e);
   }
 };
+
+exports.patchTechnician = async (req, res, next) => {
+  const updates = req.body;
+  const id = req.params.user_id;
+  try {
+    await updateTechnician(id, updates)
+    const updatedTechnician = await findTechnician(id);
```

Comment:
do we have to use a `findTechnician` call here? couldn't we just do `const updatedTechnician = await updateTechnician(id, updates);` and have that method return our patched technician?

Reply:
it doesn't send back the updated user. It sends back 
{ "acknowledged: true,
"modifiedCount": 1,
"upsertedId": null,
"upsertedCount": 0,
"matchedcount": 1
}

I wasn't sure if we needed to send back the full new Technician Data but I did it anyway for now.